core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -300,6 +300,19 @@ public ClientFactoryBuilder tlsCustomizer(Consumer<? super SslContextBuilder> tl
         return this;
     }
 
+    /**
+     * Allow the bad cipher suites listed in
+     * <a href="https://tools.ietf.org/html/rfc7540#appendix-A">RFC7540</a> for TLS handshake.
+     *
+     * <p>Note that the bad cipher suites do not meet minimum security requirements.
```

Comment:
I'm not sure what 'meet minimum security requirements' means clearly. We should convey the message that explicitly discourages its use.

How about:

    <p>Note that enabling this option increases the security risk of your connection.
    Use it only when you must communicate with a legacy system that does not support
    secure cipher suites. See ... for more information